assets/js/blocks/collection-filters/inner-blocks/price-filter/frontend.ts

```diff
@@ -3,14 +3,15 @@
  */
 import { store, navigate } from '@woocommerce/interactivity';
 import { formatPrice, getCurrency } from '@woocommerce/price-format';
+import { HTMLElementEvent } from '@woocommerce/types';
 
 /**
  * Internal dependencies
  */
-import { ActionProps, StateProps } from './types';
+import { PriceFilterState } from './types';
 
-const getHrefWithFilters = ( { state }: StateProps ) => {
-	const { minPrice, maxPrice } = state.filters;
+const getHrefWithFilters = ( state: PriceFilterState ) => {
+	const { minPrice = 0, maxPrice = 0, maxRange = 0 } = state;
```

Comment:
I don't really like what we have to do with typing the store right now.

Because we declare a lot of the state in the PHP side, these properties are technically able to be undefined. If we define them in the state client side then they will be overwritten which is not desirable.

To get around this I do leave them as possibly undefined but assign them a default value when destructuring the state.